Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart for Docker Registry
 name: docker-registry
-version: 2.2.2
+version: 2.2.3
```

Comment:
Still has the version bump, once this is removed it can be merged

Reply:
missed it, done